README.md

```diff
@@ -606,29 +670,56 @@
     }
     ```
 
-    **[[⬆]](#TOC)**
+  - Prefixing your comments with `FIXME` or `TODO` helps other developers quickly understand if you're pointing out a problem that needs to be revisited, or if you're suggesting a solution to the problem that needs to be implemented. These are different than regular comments because they are actionable. The actions are `FIXME -- need to figure this out` or `TODO -- need to implement`.
```

Comment:
Also, it isn't the syntax that's important so much as the comment prefixed with FIXME or TODO